doc/modules/language-guide/pages/motoko-introduction.adoc

```diff
@@ -95,7 +95,7 @@ We do not show an example of this "`notify`" pattern above, since it uses callba
 To address the need for clarity and simplicity, {proglang} adopts the increasingly-common program constructs `async` and `await`, which afford the programmer a _structured_ language for describing potentially-complex asynchronous dependency graphs.
 
 The link:language-manual{outfilesuffix}#exp-async[async] syntax introduces futures. A future value
-represents a _"`promise`" of a result that will be delivered, asynchronously, sometime in the future_ (not shown in the first example above).
+represents a _" `promise` " of a result that will be delivered, asynchronously, sometime in the future_ (not shown in the first example above).
```

Comment:
No idea how thus is supposed to come out... @crusso I'll leave this to you :-)

Reply:
When I looked at the markdown syntax , the below rendered output made most sense to me. 
<img width="1001" alt="Screenshot 2021-12-07 at 4 01 51 AM" src="https://user-images.githubusercontent.com/20852629/144941844-7523c072-eb0e-41da-8328-aa2f4f1c9ce3.png">

 So I've edited this doc to render a code quote as per GitHub Markdown rules. 
<img width="996" alt="Screenshot 2021-12-07 at 4 04 43 AM" src="https://user-images.githubusercontent.com/20852629/144942095-bcfb4818-1576-4722-968f-5a0de3f7ed66.png">

Refer : https://github.com/christyjacob4/motoko/blob/patch-2/doc/modules/language-guide/pages/motoko-introduction.adoc#syntactic-forms-async-and-await

Reply:
```suggestion
represents a _promise_ of a result _that will be delivered, asynchronously, sometime in the future_ (not shown in the first example above).
```